transforms/transforms.go

```diff
@@ -262,3 +262,49 @@ func GroupBy(identifier RowIdentifier) optimus.TransformFunc {
 		return nil
 	}
 }
+
+// RowFilter is meant to return `true` if a section is meant to be filtered out.
+type RowFilter func(optimus.Row) bool
+
+// passthroughTable is a super simple optimus table that just passes rows through
+type passthroughTable struct {
+	c chan optimus.Row
+}
+
+func (p passthroughTable) Rows() <-chan optimus.Row {
+	return p.c
+}
+func (p passthroughTable) Err() error {
+	return nil
+}
+func (p passthroughTable) Stop() {}
+
+// BypassTransforms effectively wraps a slice of transform funcs with a gate to conditionally
+// apply the transforms only if they match the filter.
+func BypassTransforms(doBypass RowFilter, optionalTransforms []optimus.TransformFunc) optimus.TransformFunc {
```

Comment:
I could've sworn you gave me the opposite feedback during sprint demos... https://github.com/Clever/optimus/pull/56/commits/4f840f63e8e29925ae52ee439ad1a7e696137542